### PR TITLE
Docs(cli): init docs for cli

### PR DESCRIPTION
**Pull request by lizzy-0323**
## Summary

Init commit for cli's doc.

### CODE REVIEW

File: docsite/docs/cli/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# obocli  
+
+##  Quick Start
+
+Obocli (ob-operator cli) is a command line tool that is compatible with [ob-operator](https://github.com/oceanbase/ob-operator). it has following features:
```
Comment (powerfooI) [inline docsite/docs/cli/quickstart.md:5]: `it` --> `It`.

File: docsite/docs/cli/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+- cluster management
```
Comment (powerfooI) [inline docsite/docs/cli/quickstart.md:7]: Recommend to capitalize the first letter.

File: docsite/docs/cli/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+- tenant management
+- backup policy management
+- component installation and update.
```
Comment (powerfooI) [inline docsite/docs/cli/quickstart.md:10]: Add dot for all lines or remove them all.

File: docsite/docs/cli/quickstart.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+Currently obocli is not released, you can build it from source code.
+
+## Build obocli from source code
+
+```bash
+make cli-build
+```
+
+if build successfully, you can find the binary file in `./bin/obocli`, you can copy it to your PATH.
```
Comment (powerfooI) [inline docsite/docs/cli/quickstart.md:23]: The first letter of a sentence should be capitalized.

**Comment by lizzy-0323** [comment]
Thanks for your comment and I have fixed them.

**Review by powerfooI — APPROVED** [review]
LGTM

**Pull request merged**